api/openapi.yaml

```diff
@@ -0,0 +1,1026 @@
+openapi: 3.0.0
```

Comment:
Moving to `packages/backend/src/assets` ?

Reply:
I think the openapi file should not contain in the `main` any api that is not supported.

Reply:
I'm not sure. `assets/ai.json` at build time and not copied to the user's disk. For `openapi.json`, it is different, as it needs to be copied to the disk, and read at runtime.

Reply:
> I think the openapi file should not contain in the `main` any api that is not supported.

OK, I'll remove the unimplemented endpoints and related definitions